dbms/src/Interpreters/ExpressionActions.h

```diff
@@ -201,7 +201,7 @@ class ExpressionActions
 
     static std::string getSmallestColumn(const NamesAndTypesList & columns);
 
-    BlockInputStreamPtr createStreamWithNonJoinedDataIfFullOrRightJoin(const Block & source_header, size_t index, size_t step, size_t max_block_size) const;
+    BlockInputStreamPtr createStreamWithScanHashMapAfterProbeIfNeeded(const Block & source_header, size_t index, size_t step, size_t max_block_size) const;
```

Comment:
Seems that this function is not used in this pr.

Reply:
Yeah, the only usage of this function has been removed also. Will remove this useless function.